go/app/main.go

```diff
@@ -159,8 +250,8 @@ func getImg(c echo.Context) error {
 	imgPath := path.Join(ImgDir, c.Param("imageFilename"))
 
 	if !strings.HasSuffix(imgPath, ".jpg") {
-		res := Response{Message: "Image path does not end with .jpg"}
-		return c.JSON(http.StatusBadRequest, res)
+		c.Logger().Error("Image path does not end with .jpg")
```

Comment:
nits: バグっていたらログに出してあげると優しいですね :+1:

```suggestion
		c.Logger().Errorf("Image path does not end with .jpg, got: %s", imgPath)
```

Reply:
[c750aad](https://github.com/dongurikoko/mercari-build-training/pull/3/commits/c750aadcb25bc9b797ac173ca2529f02e3f8597f)にて修正しました！